OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseAnnouncementsActivity.java

```diff
@@ -43,10 +44,12 @@ protected void onCreate(Bundle savedInstanceState) {
                 .getSerializable(Router.EXTRA_COURSE_DATA);
 
         //check courseData again, it may be fetched from local cache
-        if ( courseData != null ) {
+        if (courseData != null) {
             activityTitle = courseData.getCourse().getName();
-
-            environment.getSegment().trackScreenView(courseData.getCourse().getName());
+            environment.getSegment().trackScreenView(
+                    ISegment.Screens.COURSE_ANNOUNCEMENTS,
+                    courseData.getCourse().getId(),
+                    null);
```

Comment:
Maybe we should define a version of `trackScreenView()` that doesn't need the action parameter, considering that we're only sending `null` in these places.